src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://127.0.0.1:3306/test?user=root";
```

Comment:
The `DB_URL` is incomplete. It should include the JDBC URL specifying the database to connect to, for example 'jdbc:mysql://localhost:3306/yourDatabaseName'.